src/database/schema/ChatSchema.ts

```diff
@@ -355,8 +403,10 @@ class ChatSchema implements BaseDbSchema {
         response?.message?.updated_at,
         JSON.stringify(response),
         response?.message?.id,
+        JSON.stringify(response?.message?.attachments),
         this.id
       ];
+      console.warn('updateReplacement', JSON.stringify(updateReplacement));
 
       await db.executeSql(updateQuery, updateReplacement);
     } catch (e) {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

> Note: This review was outside of the patch, so it was mapped to the patch with the greatest overlap. Original lines [397-408]

The SQL query has been updated to include `attachment_json`, but there's no check to ensure that `attachment_json` is a valid JSON string. Consider adding a check before executing the query.
```diff
-        SET status = ?, created_at = ?, updated_at = ?, raw_json = ?, id = ?, attachment_json = ?
-        WHERE id = ?;`;
+        SET status = ?, created_at = ?, updated_at = ?, raw_json = ?, id = ?, attachment_json = CASE WHEN json_valid(?) THEN ? ELSE NULL END
+        WHERE id = ?;`;

-      const updateReplacement = [
-        response?.message?.status,
-        response?.message?.created_at,
-        response?.message?.updated_at,
-        JSON.stringify(response),
-        response?.message?.id,
-        JSON.stringify(response?.message?.attachments),
-         this.id
-       ];
+      const updateReplacement = [
+        response?.message?.status,
+        response?.message?.created_at,
+        response?.message?.updated_at,
+        JSON.stringify(response),
+        response?.message?.id,
+        JSON.stringify(response?.message?.attachments || []),
+        this.id
+      ];
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
Yes attachment_json is TEXT